src/SourceBuild/content/build.sh

```diff
@@ -33,12 +34,13 @@ usage()
   echo ""
 
   echo "Advanced settings:"
-  echo "  --build-tests                   Build repository tests. May not be supported with --source-only"
+  echo "  --build-repo-tests              Build repository tests. May not be supported with --source-only"
   echo "  --ci                            Set when running on CI server"
   echo "  --clean-while-building          Cleans each repo after building (reduces disk space usage, short: -cwb)"
   echo "  --excludeCIBinarylog            Don't output binary log (short: -nobl)"
   echo "  --prepareMachine                Prepare machine for CI run, clean up processes after build"
   echo "  --use-mono-runtime              Output uses the mono runtime"
+  echo "  --test-no-build                 Run tests without building when invoked with --test"
```

Comment:
This option doesn't seem necessary with the introduction of the test.proj.  Looking at the CI, this option isn't specified and the main build.proj is not built.  There doesn't seem a way to build and test with one exec.

Reply:
Yes, `--test-no-build` was originally needed but I then changed direction by introducing a tests.proj. I can remove the switch from this PR but I would then just introduce it in my next PR which will make it possible to build and test the VMR via one command. Thoughts?

Reply:
I think it makes sense to remove it here and include it in your next PR with the supporting changes.  It makes for more cohesive PRs IMO.

Reply:
Removed with the latest commit